Makefile

```diff
@@ -1,261 +1,66 @@
-REPO_ROOT=$(abspath $(dir $(lastword $(MAKEFILE_LIST))))
+DOCKER=1
 
-# Basenames
-CONTROLLER=controller
-IMPORTER=importer
-F_TEST=datastream-test
-U_TEST=unit-test
-U_TEST_CONTROLLER=unit-test-controller
-U_TEST_IMAGE=unit-test-image
+.PHONY: build build-controller build-importer \
+		docker docker-controller docker-cloner docker-importer \
+		test test-functional test-unit \
+		publish \
+		vet \
+		format
 
-# Binary Path
-BIN=$(REPO_ROOT)/bin
-CONTROLLER_BIN=import-controller
-IMPORTER_BIN=importer
-F_TEST_BIN=$(BIN)/$(F_TEST)
-U_TEST_BIN=$(BIN)/$(U_TEST)
-U_TEST_BIN_CONTROLLER=$(BIN)/$(U_TEST_CONTROLLER)
-U_TEST_BIN_IMAGE=$(BIN)/$(U_TEST_IMAGE)
+all: clean docker
```

Comment:
do you want _docker_ to be the recipe name here? I liked _image_ from the orig Makefile, or maybe something with _container_ in the name, ie not docker specific? What if later we want cri-o containers built?